packages/paste-website/src/pages/content/get-support.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: Get support
+description: Self-service tips and how to access UX Writer support from Caroline (London, GMT).
+---
+
+import {graphql} from 'gatsby';
+import {Link} from 'gatsby';
+import {Blockquote} from '../../components/Blockquote';
+import {Callout, CalloutTitle, CalloutText} from '../../components/callout';
+import {DoDont, Do, Dont} from '../../components/DoDont';
+import Changelog from '@twilio-paste/icons/CHANGELOG.md';
+
```

Comment:
We could use `Anchor`  in place of the normal `<a>` tags in this file. Import it above like this: 

`import {Anchor} from '@twilio-paste/anchor';`

Then replace your `<a>` tags with:

`<Anchor href="ADD_URL_HERE">`

Reply:
Using the markdown link syntax should do the trick too, right?

`[I'm the link text](https://www.imtheurl.com)`

Reply:
Yup. Good call.

Reply:
Having trouble making the []() syntax work inside a component (CallOut) for an external link  - have kept the <a> tags for now, will that be ok?

Reply:
When inside of a component, the markdown syntax doesn't work properly. If you're up for it, you can try the `Anchor` component inside the Callout. That will handle the external linking for you.

Reply:
Got it, thanks - seems to have worked ok?